Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1197,7 +1197,7 @@ def genS(fragment,howMuch):
 hiDefaults2023_ppReco_approxClusters = {'--conditions':'auto:phase1_2023_realistic_hi', '--era':'Run3_pp_on_PbPb_approxSiStripClusters_2023'}
 upcDefaults2023 = {'--conditions':'auto:phase1_2023_realistic_hi', '--era':'Run3_2023_UPC'}
 hiDefaults2024_ppReco = {'--conditions':'auto:phase1_2024_realistic_hi', '--era':'Run3_pp_on_PbPb'}
-upcDefaults2024 = {'--conditions':'auto:phase1_2024_realistic_hi', '--era':'Run3_UPC'}
+upcDefaults2024 = {'--conditions':'141X_mcRun3_2024_realistic_HI_Candidate_2024_05_30_15_21_51', '--era':'Run3_UPC'}
```

Comment:
eventually this should be re-absorbed in the regular `auto:phase1_2024_realistic_hi` when an updated one is available.

Reply:
> I would suggest to replace it [in autoCond](https://github.com/cms-sw/cmssw/blob/master/Configuration/AlCa/python/autoCond.py#L101) (rather than here) for the tests, so that the new GT is tested wherever is used. Then, once we are happy with it, a versioned GT will be created, and replaced over there once for all.

Addressed in b5cab189f7494bff633bee922250da290fef6a9f